cli_reference/kn-cli-tools.adoc

```diff
@@ -0,0 +1,44 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="kn-cli-tools"]
+= Knative CLI (`kn`) for use with OpenShift Serverless
+:context: kn-cli-tools
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+`kn` extends the functionality of the `oc` or `kubectl` tools to enable interaction with Knative components on {product-title}.
```

Comment:
I would say that `kn` don't extend functionality of `oc` or `kubectl`, rather that simplify knative related tasks. Extends in my mind means I can do all that `oc` and `kubectl` let me, and something above that.

`oc` extends functionality of `kubectl`, that's true. `kn` is different.